### ISSUE TITLE
site 2018 - review

### ISSUE THREAD

**Issue by ghost** [PHPVale/phpvale.com.br#10]
### new implementations

- [ ] add jobs section - below events section
- [ ] add blog section - below jobs section
- [x] add events page
- [x] implement font-awesome (4.7.0) and jquery (3.3.1) inside the project

**Comment by Diego-Brocanelli** [comment]
Olá, @denilsonraimundo 

Atualmente o site do PHPVale tem uma hospedagem, ou está no Github pages?

Seria possível obtermos uma hospedagem?

**Comment by Diego-Brocanelli** [comment]
Podemos ver empresas de hosting para apoiar nossa iniciativa.

**Comment by ghost** [comment]
Olá @Diego-Brocanelli,

Hoje o site está em uma hospedagem própria do @deoliveiralucas, mas podemos sim entrar em contato com algumas empresas para nos ajudar nisso.

**Comment by Diego-Brocanelli** [comment]
Bacana, vamos estruturar nossa organização no GitHub, pois vamos utilizar ele como portfólio de apresentação as empresas.

**Comment by Diego-Brocanelli** [comment]
O que acham de usar o Wordpress como plataforma para o site? Pois com isso teremos toda uma estrutura pronta para uso.

**Comment by lucasxciv** [comment]
@Diego-Brocanelli hoje o site é basicamente uma página estática, pois a frequência de atualização é bem baixa. Quando precisamos atualizar o site, nós apenas subimos uma atualização de JSON/HTML na branch master do [repositório](https://github.com/PHPVale/phpvale.com.br) que o site já é atualizado, essa forma tem nos atendido bem até o momento pelo conteúdo que temos. Acredito que a ideia do CMS seja legal para o caso de termos mais conteúdo no site, como artigos, vagas, etc. O que acha?

**Comment by Diego-Brocanelli** [comment]
Concordo, com a baixa demanda de conteúdo a necessidade de uma grande estrutura é baixa, porém para tocarmos algumas ideias será necessário essas mudanças, e no caso já mencionei de cara um WordPress para não termos trabalho reinventando a roda.

Vamos estrutura  outras ideias e assim que elas estiverem maduras voltamos a debater sobre a plataforma.

**Comment by lucasxciv** [comment]
Legal! Isso é verdade, nesse caso acho bacana a ideia do Wordpress também; a princípio, se for o caso, poderíamos até ver de subir um Wordpress e utilizar a API dele nesse site que temos atualmente, talvez seja até mais simples, mas como disse, podemos ver melhor isso quando as ideias estiveram mais maduras. Inclusive, acho que podemos ir deixando algumas ideias registradas por aqui. :)
